src/vanta_ledger/routes/companies.py

```
@@ -0,0 +1,76 @@
+from ..auth import AuthService
+from ..utils.validation import input_validator
+from ..config import settings
+from ..database import get_postgres_connection
+
+router = APIRouter(prefix="/companies", tags=["Companies"])
+
+@router.get("/")
+async def get_companies(
+    page: int = 1,
+    limit: int = 20,
+    current_user: dict = Depends(AuthService.verify_token)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)